api/alembic/versions/9a5bb047ae19_import_revised_hfi_calc_data.py

```diff
@@ -0,0 +1,70 @@
+"""Import revised HFI calc data
+
+Revision ID: 9a5bb047ae19
+Revises: 1161fab52ef5
+Create Date: 2022-02-09 11:14:13.773005
+
+"""
+from alembic import op
+import json
+
+
+# revision identifiers, used by Alembic.
+revision = '9a5bb047ae19'
+down_revision = '1161fab52ef5'
+branch_labels = None
+depends_on = None
+
+
+def get_fuel_type_id(fuel_type):
+    # Helper function to get fuel_types.id
+    conn = op.get_bind()
+    cursor = conn.execute(f"SELECT id FROM fuel_types WHERE abbrev = '{fuel_type}'")
+    result = cursor.fetchall()
+    return result[0][0]
+
+def upgrade():
+    conn = op.get_bind()
+
+    # Load stations data
+    with open('alembic/versions/9a5bb047ae19_import_revised_hfi_calc_data.json') as stations_data_file:
+        stations_data = json.load(stations_data_file)
+
+    # First delete all data from planning_areas and planning_weather_stations - going to overwrite it
+    # with updated data
+    op.execute('DELETE FROM planning_weather_stations')
+    op.execute('DELETE FROM planning_areas')
```

Comment:
If my PR beats yours, you're going to have some more tables you have to clear out.